packages/common-all/src/types/store.ts

```diff
@@ -14,6 +14,9 @@ export type FindNoteOpts = {
   fname?: string;
   // If vault is provided, filter results so that only notes with matching vault is returned
   vault?: DVault;
+  // Find notes with links referencing to passed in note props value.
+  // E.g. if linkedToNote = "{ fname: foo.123, ... }", return all notes pointing to this note
+  linkedToNote?: NotePropsMeta;
```

Comment:
I guess a similar discussion to last time, but I'm not sure how I feel about these params - it doesn't seem to be too compatible with the fname/vault parameters above. What happens if someone sets both fname and linkedToNote? 

Should this be called backlinksToNote? I think there's also a typo in the description, "referencing _to_ passed in"?

Reply:
Long term for this method, a note has to fulfill all requirements to be returned (like a sql statement where the conditions are AND together).

On brighter news, I just realized that this is the same thing as a note's backlinks (`NoteUtils.getNotesWithLinkTo` can also be replaced eventually). Currently it won't work b/c backlinks is broken, but for the new engine, I'll use the backlinks prop so we won't need `linkedToNote`